mods/ra2/rules/defaults.yaml

```diff
@@ -392,6 +392,7 @@
 		Voice: Attack
 	ActorLostNotification:
 	BodyOrientation:
+		QuantizedFacings: 32
```

Comment:
As i said, Voxels at here using `32` and it works with no problem but i'll test `0` too.